FirebaseAppCheck/Tests/Unit/DebugProvider/FIRAppCheckDebugProviderFactoryTests.m

```diff
@@ -31,6 +31,7 @@ - (void)testCreateProviderWithApp {
   options.APIKey = @"api_key";
   options.projectID = @"project_id";
   FIRApp *app = [[FIRApp alloc] initInstanceWithName:@"testInitWithValidApp" options:options];
+  app.dataCollectionDefaultEnabled = NO;
```

Comment:
Maybe a bit tedious, but do you think it's worth adding a comment on why this line is here? here and elsewhere

Reply:
Good idea, added notes throughout.